addons/mail/static/src/models/thread/thread.js

```diff
@@ -1984,6 +1986,12 @@ registerModel({
             compute: '_computeHasSeenIndicators',
             default: false,
         }),
+        /**
+         * States whether current user can add other followers to this thread.
+         */
+        hasWriteAccess: attr({
+            default: false,
+        }),
```

Comment:
For the doc: the field is not specifically related to followers, but to the rights to edit the thread in general. It would be ok to specify this includes adding followers, but it should mention in priority its general meaning.